README.md

```diff
@@ -2,6 +2,6 @@
 
 Howdy. We made this as a quick place to get recommendations for "how to start a new project"
 
-Right now, we're using Next.JS, TypeScript, and Tailwind.
+Right now, we're using Astro and Tailwind.
```

Comment:
This line is for describing the stack, not the stack of the website. Here's my suggestion:

```suggestion
Right now, we're using the [T3 Stack](https://create.t3.gg)
```

Reply:
Oh, I see. Thanks for feedback!